remote_attestation/rust/src/message.rs

```diff
@@ -23,6 +23,9 @@ use bincode;
 use serde::{Deserialize, Serialize};
 use serde_big_array::BigArray;
 
+/// Maximum message size is set to be equal to 1Kb.
```

Comment:
(nit) 1024 is 1KiB, not 1Kb

Ki = 1024
k = 1000

also

b = bit
B = byte

https://en.wikipedia.org/wiki/Byte#Multiple-byte_units

Reply:
Done